test/parallel/test-runner-run.mjs

```diff
@@ -1,9 +1,12 @@
+// Flags: --expose-internals
```

Comment:
Couldn't this also be done from the non-internal util?

Reply:
@RedYetiDev yeah I thought I did it already, I don't know where that commit went 😄 fixed !